redash/handlers/query_results.py

```diff
@@ -201,8 +157,13 @@ def post(self, query_id):
                                 any cached result, or executes if not available. Set to zero to
                                 always execute.
         """
-        params = request.get_json(force=True)
-        parameters = params.get('parameters', {})
+        if request.is_json:
+            params = request.get_json(force=True)
+            parameter_values = params.get('parameters')
+        else:
+            params = request.args
+            parameter_values = collect_parameters_from_request(params)
```

Comment:
Are you sure we _have_ to support this for this API?

Reply:
No, it's gone.